include/alp/algorithms/cholesky.hpp

```diff
@@ -136,7 +146,8 @@ namespace alp {
 				// LL[ k+1: , k+1: ] -= v*v^T
 				auto Lprim = get_view( LL, utils::range( k + 1, n ), utils::range( k + 1, n ) );
 
-				auto vvt = outer( v, ring.getMultiplicativeOperator() );
+				auto vstar = conjugate( v );
+				auto vvt = outer( vstar, ring.getMultiplicativeOperator() );
```

Comment:
Wouldn't this compute $\bar{v}v^\*$ instead of $vv^\*$?

Reply:
Yes, because this is the upper triangular version of algorithm.